src/index.ts

```diff
@@ -0,0 +1,109 @@
+import os from 'os';
+import {
+  CosmiconfigResult,
+  ExplorerOptions,
+  Loaders,
+  LoaderEntry,
+  SyncLoader,
+  AsyncLoader,
+} from './types';
+import { createExplorer } from './createExplorer';
+import * as loaders from './loaders';
+
+function identity<T>(x: T): T {
+  return x;
+}
+
+// TODO: can it be undefined to disable a loader?
+interface RawLoaders {
+  [key: string]: LoaderEntry | SyncLoader | AsyncLoader;
+}
```

Comment:
Is it expected for a user to be able to remove a file type loader by providing an undefined/null value value? If so I'll need to add void here. (I'm not requesting this, just want to have the types be correct)

Reply:
No, that's not documented or expected. (If someone doesn't want to use a default loader, they should adjust `searchPlaces` to avoid searching for files with that extension.)